geomesa-tools/bin/install-imageio-ext.sh

```diff
@@ -1,4 +1,4 @@
-#!/usr/bin/env bash
+#! /usr/bin/env bash
```

Comment:
do these spaces matter? I'd rather revert them, especially for these files which aren't otherwise modified.